components/KiboMyProfile.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="profile">
+    <div class="profile-list">
+      <div class="profile-list__header">Customer Name</div>
+      <div class="profile-list__subheader">
+        <div>{{ user.firstName }}</div>
+        <div class="profile-list__link" @click="$emit('click:edit-user-name', user)">Edit</div>
+      </div>
+    </div>
+    <div class="profile-list">
+      <div class="profile-list__header">Email</div>
+      <div class="profile-list__subheader">
+        <div>{{ user.emailAddress }}</div>
+        <div class="profile-list__link" @click="$emit('click:edit-user-email', user)">Edit</div>
+      </div>
+    </div>
+    <div class="profile-list">
+      <div class="profile-list__header">Phone Number</div>
+      <div class="profile-list__subheader">
+        <div>{{ user.phoneNumber }}</div>
+        <div class="profile-list__link" @click="$emit('click:edit-user-phone', user)">Edit</div>
+      </div>
+    </div>
+    <div class="profile-list">
+      <div class="profile-list__header">Password</div>
+      <div class="profile-list__subheader">
+        <div>{{ user.password }}</div>
+        <div class="profile-list__link" @click="$emit('click:edit-user-password', user)">Edit</div>
```

Comment:
I think we  should use localization for Customer Name, Email, Phone Number, Password and Edit